src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -739,7 +737,17 @@ def _create(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key
                 "value": client_secret
             }
         }
-    return _invoke_deployment(resource_group_name, deployment_name, template, params, validate, no_wait)
+
+    # Due to SPN replication latency, we do a few retries here
```

Comment:
aks create code calls containerservice sdk client
acs create code calls arm deployment client

So I cannot consolidate them now. I need to duplicate the code a bit.

Reply:
It's ok. 

But if you really want to reuse code, what you can do is pass a reference to the function into the reusable code of retrying.

Just a suggestion, not blocking.